### PR TITLE
Update Poplar1 syntax to match VDAF definition

### PR DESCRIPTION
**Pull request by schoppmp**

### CODE REVIEW

File: draft-irtf-cfrg-vdaf.md
Comment (cjpatton) [inline draft-irtf-cfrg-vdaf.md:2268]:
```suggestion
`beta`, one for each "level" `1 <= L <= BITS`. The key generation algorithm generates two
```

File: draft-irtf-cfrg-vdaf.md
Comment (cjpatton) [inline draft-irtf-cfrg-vdaf.md:2269]:
```suggestion
IDPF keys, one for each Aggregator. When evaluated at level `L` and index `0 <= x < 2^L`, each
```

File: draft-irtf-cfrg-vdaf.md
Comment (cjpatton) [inline draft-irtf-cfrg-vdaf.md:2283]: Ack, feel free to remove my comment.

File: draft-irtf-cfrg-vdaf.md
Comment (cjpatton) [inline draft-irtf-cfrg-vdaf.md:2292]: nit: Remove colon (":") and put the function definition on a single line. https://github.com/cfrg/draft-irtf-cfrg-vdaf/blob/main/draft-irtf-cfrg-vdaf.md?plain=1#L476

Here and below.

File: draft-irtf-cfrg-vdaf.md
Comment (cjpatton) [inline draft-irtf-cfrg-vdaf.md:2306]: The reference implementation begins indexing the levels at `0` rather than `1`. I don't care which we go with, as long as we're consistent. If you want to change this, either file an issue to fix the reference implementation or fix the reference implementation in this PR.

File: draft-irtf-cfrg-vdaf.md
Comment (cjpatton) [inline draft-irtf-cfrg-vdaf.md:2316]:
```suggestion
  associated constants as `Field` in {{field}}.
```

File: draft-irtf-cfrg-vdaf.md
Comment (cjpatton) [inline draft-irtf-cfrg-vdaf.md:2374]:
```suggestion
  (key_0, key_1) = Idpf.gen(alpha, beta[:-1], beta[-1])
```

**Review by cjpatton — COMMENTED** [review]
This is definitely moving in the right direction. One open question before we merge: Should `level` be in range `[1, BITS]` or `[0, BITS)`? The latter is used in the reference code, if I remember right.

File: draft-irtf-cfrg-vdaf.md
Comment (schoppmp) [reply draft-irtf-cfrg-vdaf.md]: Done.

File: draft-irtf-cfrg-vdaf.md
Comment (schoppmp) [reply draft-irtf-cfrg-vdaf.md]: Done.

**Review by cjpatton — APPROVED** [review]
Approving to unblock this merge so that you can go on PTO with a clean plate. This is a great start!

Would you mind squashing before merging?

**Comment by cjpatton** [comment]
For the record, this partially addresses #84.

**Pull request merged**
